packages/checkout-extension/src/ExtensionService.ts

```diff
@@ -104,4 +59,37 @@ export class ExtensionService {
 
         return Boolean(extension);
     }
+
+    private registerHandlers(extension: Extension): void {
+        const handlerProps = {
+            checkoutService: this.checkoutService,
+            dispatch: this.dispatch,
+            extension,
+        };
+
+        if (!this.handlers[extension.id]) {
+            this.handlers[extension.id] = [];
+        }
+
+        const isCommandHandler = <T extends keyof ExtensionCommandMap>(
+            type: T,
+            handler: CommandHandler<any>,
```

Comment:
```suggestion
            handler: CommandHandler<unknown>,
```

Reply:
TS does not allow it to be unknown and throws an error:
```
TS2344: Type 'unknown' does not satisfy the constraint 'keyof ExtensionCommandMap'
```